sdk/servicebus/azure-messaging-servicebus/pom.xml

```diff
@@ -52,7 +52,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-amqp</artifactId>
-      <version>1.7.0-beta.1</version> <!-- {x-version-update;beta_com.azure:azure-core-amqp;dependency} -->
+      <version>1.7.0-beta.2</version> <!-- {x-version-update;unreleased_com.azure:azure-core-amqp;dependency} -->
```

Comment:
The master branch just updated to 1.7.0-beta.3. You might want to update this to beta.3 as well?